edx_proctoring/views.py

```diff
@@ -852,15 +852,16 @@ def make_review(self, attempt, data, backend=None):
                     review_status=review.review_status
                 )
 
-        # updating attempt status will trigger workflow
-        # (i.e. updating credit eligibility table)
-
-        update_attempt_status(
-            attempt['proctored_exam']['id'],
-            attempt['user']['id'],
-            attempt_status,
-            raise_if_not_found=False
-        )
+        if not attempt.get('is_archived', False):
```

Comment:
@nedbat it turns out that archived attempts are used a lot. I restored the check, to prevent status changes on archived attempts.
Every save to the attempt table duplicates the data in the attempt history table. We should revisit this to figure out whether it's necessary to keep all of this duplicated data around.